### ISSUE TITLE
About Generate Image from GT Representation resault

### ISSUE THREAD

**Issue by Jasonhyw** [LTH14/rcg#12]
Dear author, in the GT-generated images, the images containing humans do not seem to produce very good results. Is there any problem with my implementation? Looking forward to your reply, thanks.

**Comment by LTH14** [comment]
Yes, human is a common problem when generating ImageNet images, especially human faces. If only generating human is with poor performance and other generation looks reasonable, your implementation should be correct.
